CHANGELOG

```diff
@@ -1,4 +1,7 @@
-* Thu Mar 07 2019 Liz Nemsick <lnemsick.simp@gmail.com> - 0.0.3
+* Fri Aug 02 2019 Robert Vincent <pillarsdotnet@gmail.com> - 0.0.5
+- Support puppetlabs/stdlib 6.x and puppet 6.x.
+
+* Thu Mar 07 2019 Liz Nemsick <lnemsick.simp@gmail.com> - 0.0.4
```

Comment:
* Multiple changelog entries are possible for a version
* It is not appropriate to change the changelog version of a release component. 0.0.3 was released: https://github.com/simp/pupmod-simp-dconf/releases/tag/0.0.3

I'll fix along with other test changes.